packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/ProxyApiRegistrar.java

```diff
@@ -60,7 +60,7 @@ void onFailure(@NonNull String methodName, @NonNull Throwable throwable) {
   }
 
   // PreviewProxyApi maintains a state to track SurfaceProducers provided by the Flutter engine.
-  @NonNull private final PreviewProxyApi previewProxyApi = new PreviewProxyApi(this);
```

Comment:
This was causing a lint error when building locally: https://wiki.sei.cmu.edu/confluence/display/java/TSM01-J.+Do+not+let+the+this+reference+escape+during+object+construction